src/types/index.ts

```diff
@@ -83,18 +91,67 @@ export interface ITaskTemplate extends ITemplate {
 
 interface ITaskTemplateData extends ITemplateData {
   type: TemplateType.Task;
-  attributes: {name: string, flux: string};
+  attributes: { name: string; flux: string };
   relationships?: {
-    [TemplateType.Label]: {data: ILabelRelationshipDataItem[]};
+    [TemplateType.Label]: { data: ILabelRelationship[] };
   };
 }
 
-interface ILabelRelationshipDataItem {
+interface ITaskTemplateIncluded extends ITemplateIncluded {
```

Comment:
can this just be ITaskTemplate?

Reply:
ITaskTemplate is already something else. 
```
export interface ITaskTemplate extends ITemplate {
  content: {
    data: ITaskTemplateData;
    included?: ITaskTemplateIncluded[];
  };
}
```

The ITaskTemplateIncluded are objects that represent the different resources that are included in the task template. For example, a Task will include labels in the Template. However, a dashboard would have labels, cells, and views

Reply:
we already have an ITaskTemplate